### PR TITLE
Aggregate fixes

### PR DESCRIPTION
**Pull request by wmarshall484**

### CODE REVIEW

**Comment by wmarshall484** [comment]
Also, change `pickle_in__*__out` to `object_in__*_out`

**Comment by wmarshall484** [comment]
@ddebrunner, I added commit [536ceec](https://github.com/IBMStreams/streamsx.topology/pull/1273/commits/536ceeca77a9ba48a9ce2e77b8143641f3ac8c0f) to no longer wrap `streamsx.runtime.identity`.

**Comment by ddebrunner** [comment]
I was hoping these changes could have been separated out, so the first set (that I thumbed up) could have been merged, and then the follow on changes handled separately in a new PR. Otherwise a PR tends to drag out with a potentially endless cycle of {changes, review, make more changes requiring an new review, ...}

BTW any reason these changes are being made directly in an IBMStreams branch and not a fork?

File: com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/Aggregate/Aggregate_cpp.cgt [hunk 1]
```diff
@@ -9,23 +9,20 @@ using namespace streamsx::topology;
...
  my $inputPort = $model->getInputPortAt(0); 
  my $window = $inputPort->getWindow();
  my $windowCppInitializer = SPL::CodeGen::getWindowCppInitializer($window,"PyObject *");
 
-
  # Select the Python wrapper function
  my $pyoutstyle = splpy_tuplestyle($model->getOutputPortAt(0));
 
- if (($pystyle eq 'dict') || ($pyoutstyle eq 'dict')) {
+ if (($pystyle eq 'dict') || ($pyoutstyle eq 'dict') || ($pystyle eq 'tuple') || ($pyoutstyle eq 'tuple')) {
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/Aggregate/Aggregate_cpp.cgt]: `pyoutstyle` can never be `tuple`.

**Comment by wmarshall484** [comment]
Removed the last commit, separating out into two PRs.

**Pull request merged**